packages/adaptive-ui-explorer/src/components/adaptive-component.ts

```diff
@@ -49,7 +49,7 @@ export class AdaptiveComponent extends FASTElement {
     @observable
     public styles?: Styles;
     protected stylesChanged(prev: Styles, next: Styles) {
-        console.log("stylesChanged", next);
+        // console.log("stylesChanged", next);
```

Comment:
remove comments?

Reply:
For this one, sure. I've been keeping many of them around in the tooling because it's often the only way to understand what's going on.